src/text.cxx

```diff
@@ -43,17 +44,33 @@ GetOffset (const char *& c)
   };
 
   static const unsigned char table_extended[] = {
+  /* Italian */
   /*  ò     ì                                                */
     0xB2, 0xAC,                                     /* UTF-8 */
+  /* Polish */
+  /*              ń     ó     ę     ć     ź     ż            */
+                0x84, 0xB3, 0x99, 0x87, 0xBA, 0xBC, /* UTF-8 */
+  /*  ą     ł     ś                                          */
+    0x85, 0x82, 0x9B,                               /* UTF-8 */
   };
   /* clang-format on */
 
   if (static_cast<unsigned char> (*c) == 0xC3)
     c++;
+  if (static_cast<unsigned char> (*c) == 0xC4)
+    c++;
+  if (static_cast<unsigned char> (*c) == 0xC5)
+    c++;
 
-  for (unsigned int i = 0; i < countof (table_accents); ++i)
-    if ((unsigned char) *c == table_accents[i])
-      return 15 + i;
+  if (GetLocale() != "pl")
+  {
+    // Do not use the 'standard' accents table with Polish locale
+    // This is required because we check only last byte of UTF-8 and some characters overlap
+    // TODO: In the future, this ugly hack should be replaced with proper UTF-8 parsing
+    for (unsigned int i = 0; i < countof (table_accents); ++i)
```

Comment:
The proper way will be done with `SDL_ttf` for the v2 of the game (I'm working on it; there are a lot of work). It doesn't matter for v1.